### ISSUE TITLE
Small UI bug

### ISSUE THREAD

**Issue by ag84ark** [AgoraIO-Community/VideoUIKit-Flutter#122]
I think this should be: 
```
widget.enableHostControl == null || widget.enableHostControl  == false
```
Now, if you put false, it still shows them, actually you can't make them disappear.

https://github.com/AgoraIO-Community/VideoUIKit-Flutter/blob/726f3ad71ed824ed5a91e4b833e380c52532f01c/lib/src/layout/floating_layout.dart#L257

https://github.com/AgoraIO-Community/VideoUIKit-Flutter/blob/726f3ad71ed824ed5a91e4b833e380c52532f01c/lib/src/layout/floating_layout.dart#L373

https://github.com/AgoraIO-Community/VideoUIKit-Flutter/blob/726f3ad71ed824ed5a91e4b833e380c52532f01c/lib/src/layout/floating_layout.dart#L450

**Comment by maxxfrazer** [comment]
@Meherdeep looks like @ag84ark is right here.

or the condition could be changed to `widget.enableHostControl != true`.

**Comment by Meherdeep** [comment]
Fixed with v1.2.0
